templates/lex_bot_kendra_master.template.yaml

```diff
@@ -148,6 +148,25 @@ Resources:
           Action:
           - sts:AssumeRole
       Path: "/"
+      Policies:
```

Comment:
This isn't needed. The LambdaFunctionIAMPolicy resource has necessary permissions, which is tied to this IAMRole.